change/@fluentui-react-components-08f8b917-6f84-41d9-a80d-ad446f36f690.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: update spinbutton to stable",
```

Should this be of type `chore`?

```suggestion
  "comment": "chore: update spinbutton to stable",
```